core-java-modules/core-java-io-conversions-3/src/test/java/com/baeldung/xlsxtocsv/ConvertToCSVUnitTest.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.xlsxtocsv;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ConvertToCSVUnitTest {
+
+    private static final String XLSX_FILE_INPUT = "src/test/resources/xlsxToCsv_input.xlsx";
+    private static final String CSV_FILE_OUTPUT = "src/test/resources/xlsxToCsv_output.csv";
+
+    @Test
+    public void givenXlsxFile_whenUsingCommonsCSV_thenGetValuesAsList() throws IOException {
```

Comment:
We can remove the `public` keyword here. By convention JUnit 5 uses package private test methods.